lib/src/over_react_test/common_component_util.dart

```diff
@@ -249,15 +264,61 @@ void _testPropForwarding(BuilderOnlyUiFactory factory, dynamic childrenFactory()
     for (var forwardingTarget in forwardingTargets) {
       Map actualProps = getProps(forwardingTarget);
 
-      // If the forwarding target is a DOM element it will should not have invalid DOM props forwarded to it.
+      // If the forwarding target is a DOM element it should not have invalid DOM props forwarded to it.
       if (isDomElement(forwardingTarget)) {
         otherProps.forEach((key, value) {
-          expect(actualProps, isNot(containsPair(key, value)));
+          expect(actualProps, isNot(containsPair(key, value)), reason: unindent('''
+            The following mock key/value pair(s) added by this test were found on 
+            a DOM component that props were forwarded to:
+            
+                $key: ${actualProps[key]}
+                 
+            This means that `addUnconsumedProps` is mistakenly being used 
+            instead of `addUnconsumedDomProps` on a DOM component.
+          '''));
         });
       } else {
         otherProps.forEach((key, value) {
           expect(actualProps, containsPair(key, value));
         });
+
+        final missingUnconsumedPropKeys = unconsumedProps.keys.where((key) => !actualProps.containsKey(key));
+        final mixinNamesOfMissingUnconsumedPropKeys = missingUnconsumedPropKeys.map(getPropKeyNamespaceFromPropKey).where((name) => name != null).toSet();
+        expect(mixinNamesOfMissingUnconsumedPropKeys, isEmpty, reason: unindent('''
```

Comment:
The above comment is still unaddressed; this should be:
```suggestion
        expect(missingUnconsumedPropKeys, isEmpty, reason: unindent('''
```

Reply:
Addressed in https://github.com/Workiva/over_react_test/pull/102/commits/7a98196fc81981ba1d0c586c0d6d896709934108